src/ProductPage.jsx

```diff
@@ -1,140 +1,72 @@
-import React, { useState, useEffect } from "react";
-import { useSearchParams } from "react-router-dom";
+
+import React, { useState } from "react";
```

Comment:
Why do you have to import `React`, I belive in older versions it was required to be done, but in newer once it is not.